pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -26,8 +26,8 @@ stdenv.mkDerivation {
   
   buildInputs = [ gmp mpfr libmpc zlib ];
   
-  # Make sure we don't strip the libraries in lib/gcc/avr.
-  stripDebugList= [ "bin" "avr/bin" "libexec" ];
+  # Make sure we don't strip the libraries in lib/gcc/avr and in avr/lib.
```

Comment:
My experience is that *.o files in avr/lib make strip fail because their format is not recognised.
Only avr-strip could possibly recognise these. I will update the patch to reflect this.
Later, it may be a good idea to strip these files with avr-strip.
@ambrop72, does it make sense to you ?

Anyway, I will update the comment.